tls_gen/cli.py

```diff
@@ -13,6 +13,8 @@ def build_parser():
                  help = "Certificate CN (Common Name)", default = socket.gethostname())
     p.add_option("--client-alt-name", dest = "client_alt_name", action = "store",
                  help = "SAN (subject Alternative Name) for the client", default = socket.gethostname())
+    p.add_option("--crl", dest = "crl", action = "store",
+                 help = "CRL for the lists", default = "example.com/fakecrl.crl.pem")
```

Comment:
Remove default value

Reply:
Could be removed if the other comment is more clear to me I think. Otherwise having an empty value in the crl of a certificate might cause problems for online checks (though of course the specified url doesnt exist so it's not a great deal either)